src/Directory.Build.props

```diff
@@ -11,7 +11,17 @@
     <NoWarn>$(NoWarn);1591</NoWarn>
     <WarningsNotAsErrors>$(WarningsNotAsErrors);1591</WarningsNotAsErrors>
 
-
+    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)\Source.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference
+      Condition="'$(UseMLCodeAnalyzer)' != 'false'"
```

Comment:
You might also want a project type condition `'$(MSBuildProjectExtension)' == '.csproj'`.

Reply:
That's a good idea

Reply:
OK I have added this as a condition to both the stylecop and project import @ericstj